{"name": "test_second_case[True-send_invalid_data-True]", "status": "passed", "steps": [{"name": "send_invalid_data", "status": "passed", "start": 1714302322829, "stop": 1714302322829}], "parameters": [{"name": "func", "value": "True"}, {"name": "step", "value": "'send_invalid_data'"}, {"name": "res", "value": "True"}], "start": 1714302322829, "stop": 1714302322829, "uuid": "a4e3675b-a730-45be-9074-a093da50a99b", "historyId": "c9b5dff89aa800b78dcd326ab9b3eaf3", "testCaseId": "25139f44ccbc0b02fc34a70a40916693", "fullName": "test_api.TestAPI#test_second_case", "labels": [{"name": "epic", "value": "Testing docker_allure"}, {"name": "story", "value": "second_test_case"}, {"name": "suite", "value": "test_api"}, {"name": "subSuite", "value": "TestAPI"}, {"name": "host", "value": "user"}, {"name": "thread", "value": "132367-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_api"}]}